tests.py

```diff
@@ -473,23 +471,48 @@ def test_post_pre_serialized_msgpack(self):
         self.assertEqual([r.code for r in response.history], [200])
 
     @testing.gen_test
-    def test_rate_limiting_behavior(self):
+    def test_rate_limiting_behavior_with_retry_after(self):
         with asynctest.mock.patch(
                 'sprockets.mixins.http.asyncio') as aio_module:
             aio_module.sleep = asynctest.CoroutineMock()
-            for rate_limit_code in {423, 429, 503}:
+            for rate_limit_code in {423, 429, 500, 503}:
```

Comment:
Add 502 since you mentioned it specifically in https://github.com/sprockets/sprockets.mixins.http/issues/34